app/Console/Commands/ScrapMoneyBox.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Panther\Client;
+
+class ScrapMoneyBox extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'scrap:moneybox';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Scrap the current moneybox value and save to DB';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $moneyboxId = env('MONEYBOX_ID');
+        $moneyboxValueSelector = env('MONEYBOX_SELECTOR');
+        $url = 'https://eskarbonka.wosp.org.pl/' . $moneyboxId;
+
+        $client = Client::createFirefoxClient();
+        // Or, alternatively use Chrome
+        // $client = Client::createChromeClient();
```

Comment:
Stretch goal, ale czy to nie mógłby być parametr do tej komendy z firefoxem jako defaultem? ;)